src/components/Sidenav/StudioSidenav/StudioSidenav.tsx

```diff
@@ -46,39 +50,52 @@ export const StudioSidenav: React.FC = () => {
     }
     return item
   })
+  const { haveVideoTabsAssetCache } = useEditVideoSheet()
+
+  const handleClick = () => {
+    if (haveVideoTabsAssetCache) {
+      openWarningDialog({
+        confirmCallback: () => {
+          setExpanded(false)
+          navigate(absoluteRoutes.viewer.index())
```

Comment:
Shouldn't we reset the cache here?

Reply:
I guess this will happen automatically because the video sheet will get closed?

Reply:
I don't think we need to do more work here. If we go to the viewer, the sheet will unmount and all the asset cache should be removed.